JDStatusBarNotification/JDStatusBarView.m

```diff
@@ -58,17 +58,45 @@ - (void)layoutSubviews;
 
   // label
   CGFloat topLayoutMargin = JDStatusBarRootVCLayoutMargin().top;
+    
+    CGFloat labelY = self.textVerticalPositionAdjustment + topLayoutMargin + 1;
+    CGFloat height = self.bounds.size.height - topLayoutMargin - 1;
+    
+    // adjust for iPhone X
+    if (topLayoutMargin > 0){
+		switch (_iphoneXSize) {
+			case JDStatusBarIphoneXSizeMini:
+				labelY = self.textVerticalPositionAdjustment + topLayoutMargin - 8.0 - 3.0; // 3.0 to adjust correctly the label
```

Comment:
let's only do the additional change here, instead of redoing the above math:
```
labelY -= 12; // adjust for smaller iPhoneX size
```

Reply:
Done